### DISCUSSION TITLE
ATI 3D RAGE GPU Emulation

### DISCUSSION THREAD

**Discussion by rryan22** [86Box/86Box#3020] [Ideas]
Happy to see support for ATI 3D RAGE graphics cards: I, II, II+DVD, IIc, Pro, XL, 128, 128 VR, 128 GL, 128 Pro, 128 Ultra, Fury MAXX

**Comment by KacperJurak468** [comment]
Proszę o GPU Ati Rage

**Reply by WhiteMagicRaven** [reply]
Here is a lot of ATI 3D RAGE documentation, including a lot of registers descriptions

https://bitsavers.trailing-edge.com/components/ati/
